### PR TITLE
Feature/string int convert

### PR DESCRIPTION
**Pull request by stepantzov**
first version.

### CODE REVIEW

File: src/training/string_reverse/StringReverseMain.java [hunk 1]
```diff
@@ -0,0 +1,19 @@
+package training.string_reverse;
```
Comment (NZelskaya) [inline src/training/string_reverse/StringReverseMain.java]: change package  name

File: src/training/string_to_int/StringIntConverter.java [hunk 2]
```diff
@@ -0,0 +1,32 @@
+package training.string_to_int;
```
Comment (NZelskaya) [inline src/training/string_to_int/StringIntConverter.java]: change package  name

File: src/checking/ShowProperties.java [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+ * Created by Ivan_Stepantsov on 7/12/2017.
+ */
+public class ShowProperties {
+    public static void main(String[] args) {
+        System.getProperties().list(System.out);
+        System.out.println(System.getProperty("user.name"));
+        System.out.println(
+                System.getProperty("java.library.path"));
+    }
+} ///:~
```
Comment (NZelskaya) [inline src/checking/ShowProperties.java]: ///:~ > what is this? and what for is this class

File: src/training/string_reverse/StringReverseMain.java [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+
+import java.util.Scanner;
+/**
+ * Created by Ivan_Stepantsov on 7/16/2017.
+ */
+public class StringReverseMain {
```
Comment (NZelskaya) [inline src/training/string_reverse/StringReverseMain.java]: why it's in this pr?

File: src/training/string_to_int/StringIntConverter.java [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+        System.out.println("Please select the operation required:\n " +
+                "1 Convert from String to Int\n 2 Convert from Int to String");
+        Scanner myScanner = new Scanner(System.in);
+        int i = myScanner.nextInt();
+        switch (i) {
+            case 1: {
+                System.out.println("Please enter String to convert to Int:");
+                Scanner enteredStringScanner = new Scanner(System.in);
+                String enteredString = enteredStringScanner.nextLine();
+                int intCaprure = Integer.parseInt(enteredString);
```
Comment (NZelskaya) [inline src/training/string_to_int/StringIntConverter.java]: Caprure ? :)

File: src/training/string_to_int/StringIntConverter.java [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+            }
+            case 2: {
+                System.out.println("Please enter Int to convert to String:");
+                Scanner enteredIntScanner = new Scanner(System.in);
+                int enteredInt = enteredIntScanner.nextInt();
+                String stringCapture = String.valueOf(enteredInt);
+                System.out.println(stringCapture);
+            }
+        }
+
```
Comment (NZelskaya) [inline src/training/string_to_int/StringIntConverter.java]: empty line :)

File: src/training/string_to_int/StringIntConverter.java [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+/**
+ * Created by Ivan_Stepantsov on 7/17/2017.
+ */
+public class StringIntConverter {
+    public static void main(String[] args) {
+        System.out.println("Please select the operation required:\n " +
+                "1 Convert from String to Int\n 2 Convert from Int to String");
+        Scanner myScanner = new Scanner(System.in);
+        int i = myScanner.nextInt();
+        switch (i) {
```
Comment (NZelskaya) [inline src/training/string_to_int/StringIntConverter.java]: Where have u lost breaks?

Comment (NZelskaya) [inline src/training/string_to_int/StringIntConverter.java]: Move this logic to separate method

File: src/checking/ShowProperties.java (hunk 3 above)
Comment (stepantzov) [inline src/checking/ShowProperties.java]: oh, it's from the book. 
i was checking some stuff on practice...

Comment (NZelskaya) [inline src/checking/ShowProperties.java]: in pr should be classes only related to task

File: src/training/string/StringReverseMain.java [hunk 8]
```diff
@@ -0,0 +1,19 @@
+package training.string;
+
+import java.util.Scanner;
+/**
+ * Created by Ivan_Stepantsov on 7/16/2017.
+ */
+public class StringReverseMain {
```
Comment (NZelskaya) [inline src/training/string/StringReverseMain.java]: А он все еще тут)

**Pull request merged**